src/App.tsx

```diff
@@ -1,17 +1,78 @@
 import { useTranslation } from 'react-i18next'
+import LangSwitcher from './components/LangSwitcher/LangSwitcher'
+import logo from './assets/logo.svg'
+import gifAdult from './assets/bg-animation.gif'
+import gifCh from './assets/bg-animation-ch.gif'
+import flowers from './assets/flowers.svg'
+import flowersMobile from './assets/flowers-2.svg'
+import './scss/sections/hero.scss'
 
 function App() {
-	const { t } = useTranslation()
+	const { t, i18n } = useTranslation()
 
 	return (
 		<>
-			<h1>Vite + React</h1>
-			<div className="card">
-				<h2 className="t__h2">{t('heroTitle')}</h2>
-			</div>
-			<div>
-				<p>ПОРУЧ ДІТИ</p>
-			</div>
+			<section className="hero">
+				{i18n.language === 'ch' && (
+					<img
+						src={window.innerWidth <= 375 ? flowersMobile : flowers}
+						className={window.innerWidth <= 375 ? 'hero__bg-flowers-mobile' : 'hero__bg-flowers'}
+					/>
+				)}
+				<img
+					src={i18n.language === 'ad' ? gifAdult : gifCh}
+					alt="Background animation"
+					className={i18n.language === 'ad' ? 'hero__bg-animation' : 'hero__bg-animation-ch'}
+				/>
+				<div className="hero__container">
+					<div className="hero__switch-and-menu-container">
+						<div className="hero__lang-switch-container">
```

Comment:
Paragraph  it's already a block element, so what else is there to do div?

Reply:
Could you please clraify wdym? Thank you!

Reply:
It's there to contain both paragraf and lang switcher